templates/python/custom-copilot-basic/env/.env.dev.user.tpl

```diff
@@ -0,0 +1,13 @@
+# This file includes environment variables that will not be committed to git by default. You can set these environment variables in your CI/CD system for your project.
+
+# Secrets. Keys prefixed with `SECRET_` will be masked in Teams Toolkit logs.
+SECRET_BOT_PASSWORD=
+{{#useOpenAI}}
+SECRET_OPENAI_API_KEY=<openai-api-key>
+SECRET_OPENAI_MODEL_DEPLOYMENT_NAME=<openai-model-development-name>
```

Comment:
@MuyangAmigo what's your opinion on having `SECRET_` prefix for non secret values? Current name works but I'm not sure whether it will cause confusions about how to name the envs under `.env.dev.user`

Reply:
`OPENAI_API-Key` should be a secret value, but I guess deployment name doesn't matter.

Reply:
Btw why we need deployment name here? Can't we just specify it in the code?